temba/formax.py

```diff
@@ -28,8 +28,21 @@ def __init__(self, request):
         context = user_group_perms_processor(self.request)
         self.org = context["user_org"]
 
+    def resolve_prefixed_url(self, url):
+        """
+        django.urls.resolve(url) is not an exact mirror for django.urls.reverse(url) as the
+        latter prefixes SCRIPT_NAME to the result, but the former does not strip the prefix
+        since Django expects the uWSGI component to do that before Django processes the URL.
+        The result is that anywhere code expects to do a resolve(reverse(url)) we will need
+        to first strip out the unwanted prefix.
+        :param str url: the result of django.urls.reverse(url)
+        :return: result of django.urls.resolve(prefix_stripped_url).
+        """
+        prefix = get_script_prefix()
+        return resolve(url[len(prefix):] if url and prefix and url.startswith(prefix) else url)
```

Comment:
Please expand this and run black and we are set.

Reply:
Expanded and run through black formatter, as requested.